Mage/src/main/java/mage/game/permanent/token/YoungHeroRoleToken.java

```diff
@@ -0,0 +1,62 @@
+package mage.game.permanent.token;
+
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.SourceMatchesFilterCondition;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.abilities.effects.common.AttachEffect;
+import mage.abilities.effects.common.continuous.GainAbilityAttachedEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.abilities.keyword.EnchantAbility;
+import mage.constants.*;
+import mage.counters.CounterType;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.ToughnessPredicate;
+import mage.target.TargetPermanent;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ * @author TheElk801
+ */
+public final class YoungHeroRoleToken extends TokenImpl {
```

Comment:
I wonder if the reminder text are always the same for each kind of role.
At which point we could add those either in the Token definition (with an interface for a getter), or in the enum grouping them.
Probably a consideration to have with the full set spoiled.

Reply:
we might be able to include the reminder text into the enum but if it were up to me we wouldn't have any reminder text in generated rules text at all anywhere